docs/checks/asset_preload.md

```diff
@@ -0,0 +1,54 @@
+# Prevent Manual Preloading of Assets (`AssetPreload`)
+
+Preloading can be a useful way of making sure that critical assets are downloaded by the browser as soon as possible for better rendering performance.
+
+Liquid provides multiple filters to [preload key resources][preload_key_resources] so they can be converted into `Link` headers automatically. This enables them to be discovered even faster, especially when combined with Early Hints that Shopify supports.
+
+## Examples
+
+The following examples contain code snippets that either fail or pass this check.
+
+### &#x2717; Fail
+
+```liquid
+<link href="style.css" rel="preload" as="style">
```

Comment:
Nit: a storefront would typically use `<link href="{{ 'style.css' | asset_url }}" rel="preload" as="style">` here. You can't really have assets as relative paths.

Reply:
Fixed ✅